CWE/events/italy.txt

```diff
@@ -54,460 +126,341 @@ country_event = {
+                       diplomatic_influence  = { who = THIS value = -200 }
+        }
+		
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
-		set_country_flag = very_moderate_ICP 
-	}
-
-	option = {
-		name = EVT_8007735_B
-		ai_chance = { factor = 0.2 }
-		any_pop = { militancy = 3 }
+                scaled_militancy = { factor = 2 ideology = conservative }
+			}	
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-		set_country_flag = very_radical_ICP 
+                scaled_militancy = { factor = 2 ideology = big_tent }
+			}	
+        any_pop = {
+                scaled_militancy = { factor = 2 ideology = liberal }
+			}		
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = conservative }
+			}	
+		any_pop = {
+                scaled_consciousness = { factor = 2 ideology = big_tent }
+			}	
+        any_pop = {
+                scaled_consciousness = { factor = 2 ideology = liberal }
+			}
+		ai_chance = { factor = 0.1 }
 	}
+   
 }
 
-#Togliatti assassination attempt
+#1948 Constitution
 country_event = {
-  id = 8007736
-  title = EVT_8007736_NAME
-  desc = EVT_8007736_DESC
-  picture = "nwo2_palmiro_togliatti"
-  fire_only_once = yes
+	id = 8007770
+	title = "The 1948 Constitution"
+	desc = EVTDESC8007770
+	picture = "nwo2_austrian_state_treaty"
+  
+	fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1948 NOT = { year = 1949 }
-		OR = { government = democracy government = hms_government }
-		has_country_flag  = very_moderate_ICP
-		NOT = { 
-			OR = { 
-				ruling_party_ideology = socialist 
-				ruling_party_ideology = communist 
-				ruling_party_ideology = communist_social 
-			} 
-		}
+		year = 1948
+		NOT = { year = 1950 }
+		
+		month = 1
+		
+		government = democracy
+		is_vassal = no
+		war = no
+		
+		has_country_flag = ITA_1948_republic_flag
 	}
 
-	mean_time_to_happen = { months = 2 }
-
-	option = {
-		name = EVT_8007736_A
-		any_pop = { consciousness = 10 }
-		ai_chance = { factor = 0.6 }
-	}
+	mean_time_to_happen = { months = 1 }
 
 	option = {
-		name = EVT_8007736_B
-		any_pop = { consciousness = 5 militancy = 5 }
+		name = "Adopt the new Constitution!"
+		prestige = 25
+		
 		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.025 value = communist }
+		scaled_consciousness = -2
+		scaled_militancy = -2
 		}
-		set_country_flag = radical_ICP
-		ai_chance = { factor = 0.2 }
-	}
+      
+		clr_country_flag = presidential_system_republic
+		clr_country_flag = semi_presidential_system_republic
+		clr_country_flag = parliamentary_system_republic
+			
+		set_country_flag = parliamentary_system_republic
 
+		ai_chance = { factor = 0.99 }
+	}
+	
 	option = {
-		name = EVT_8007736_C
-		any_pop = { militancy = 10 }
+		name = "Strive for a different constitution!"
```

Comment:
This option should select another republic type, possibly presidential so as to block the generic event from firing when then allows parliamentary to be selected again.